dvc/command/imp.py

```diff
@@ -1,11 +1,7 @@
 import os
 
-try:
-    from urlparse import urlparse
-except ImportError:
-    from urllib.parse import urlparse
-
 import dvc.logger as logger
+from dvc.utils import urlparse
```

Comment:
What I've seen before is creating a compatibility module and move all the logic that supports python 2/3 compat right there: https://github.com/requests/requests/blob/master/requests/compat.py

It is great to have it at least on `utils` :+1:

Reply:
That is actually much more reasonable and future-proof. I'll move it to compat.py in a sec. Thanks!